http-builder-ng-core/src/test/groovy/groovyx/net/http/tk/HttpGetTestKit.groovy

```diff
@@ -446,16 +446,16 @@ abstract class HttpGetTestKit extends HttpMethodTestKit {
     @Unroll 'success/failure handler with Closure (#code)'() {
         setup:
         ersatzServer.expectations {
-            get('/handling').called(2).responds().code(code)
+            get('/handling').called(2).responds().code(code).content(OK_TEXT, TEXT_PLAIN).code(code)
```

Comment:
`.code(code)` matcher is applied twice - not a big deal since its the same thing, but just duplication.